libraries/gpu/src/gpu/Texture_ktx.cpp

```diff
@@ -65,15 +65,14 @@ struct GPUKTXPayload {
     }
 
     bool unserialize(const Byte* data, size_t size) {
-        if (size != SIZE) {
```

Comment:
the problem was that this size changed, so version 1 ktx files weren't getting parsed correctly